src/components/AddNewCharacterForm/AddNewCharacterForm.js

```diff
@@ -0,0 +1,233 @@
+import { Button, Container, Grid, makeStyles } from "@material-ui/core"
+import { useForm } from "react-hook-form"
+import { useDispatch } from "react-redux"
+import { nanoid } from "@reduxjs/toolkit"
+
+import { addCharacter } from "@/reducers/characterReducer"
+
+import ControlledInputField from "../ControlledInputField"
+import ControlledSelectInputField from "../ControlledSelectInputField/ControlledSelectInputField"
+
+const useStyles = makeStyles((theme) => ({
+  title: {
+    borderBottom: "1px solid grey",
+    padding: theme.spacing(1),
+    fontWeight: theme.typography.fontWeightBold,
+  },
+
+  body: {
+    padding: "15px",
+    display: "flex",
+    flexDirection: "column",
+    gap: theme.spacing(2),
+    width: "30vw",
```

Comment:
I see you using "vw" units too much for my liking, we only use them in rare cases, we normally stick to using percentages instead